ksp_plugin_adapter/localization/en-us.cfg

```
@@ -75,7 +75,9 @@ Localization {
     #Principia_ReferenceFrameSelector_Description_BodyCentredNonRotating = The centre of <<A:1>> is fixed.\nIts equator is horizontal.\nThe reference frame does not rotate.
     #Principia_ReferenceFrameSelector_Description_BodyCentredParentDirection = The centre of <<A:1>> is fixed.\nThe plane of its orbit around <<A:3>> is horizontal.\nThe reference frame rotates with the orbit, fixing the direction of <<A:3>>.
     #Principia_ReferenceFrameSelector_Description_BodySurface = <<AC:1>> is fixed.\nIts equator is horizontal.\nThe reference frame rotates with <<A:1>>.
-    #Principia_ReferenceFrameSelector_Description_RotatingPulsating = The centres of <<A:3>> and <<A:1>> are fixed.\nThe plane of their orbit is horizontal.\nThe reference frame rotates with the orbit and pulsates, fixing the centres of both bodies.
+    #Principia_ReferenceFrameSelector_Description_RotatingPulsating = The centres of <<A:3>> and <<A:1>> are fixed.\nThe plane of their orbit is horizontal.\nThe reference frame rotates with the orbit and pulsates to keep the <<nom:3>>–<<nom:1>> distance constant.
```

```suggestion
    #Principia_ReferenceFrameSelector_Description_RotatingPulsating = The centres of <<A:3>> and <<A:1>> are fixed.\nThe plane of their orbit is horizontal.\nThe reference frame rotates and pulsates with the orbit of <<A:3>> such that the direction of <<A:3>> and the <<nom:3>>–<<nom:1>> distance are both fixed.
```
Perhaps its similarity to <secondary>-<primary> orbit should be emphasized?

---

This is tricky.
Normally the descriptions are:
1. Fixed point
2. Reference plane
3. Reference direction within that plane.

However, by giving two fixed points, we have already determined everything but the reference frame; saying that it fixes the distance is already redundant (but pulsating is weird, so I called it out). The reference direction is even more redundant.

---

Agreed. I'm just not sure if the implications are entirely obvious to the average user.